theano/tensor/nnet/nnet.py

```diff
@@ -384,6 +384,7 @@ def c_code(self, node, name, inp, out, sub):
             }
         }
 
+        #pragma omp parallel for schedule(static, 32)
```

Comment:
thanks for the timing. I don't understand why it should work without inheriting from OpenMPOp. Do you? Without that, we should not pass the -fopenmp parameter to g++, so it should ignore the pragma.

Which compiler do you use? On which OS?

This PR still need to inherit from OpenMPOp. Can you do that change?

thanks